website/source/guides/securing-nomad.html.md

```diff
@@ -431,6 +431,48 @@ Jobs running in the cluster will _not_ be affected and will continue running
 throughout the switch as long as all clients can restart within their heartbeat
 TTL.
 
+## Changing Nomad certificates on the fly
```

Comment:
Let's make this the main "upgrading to tls" content, and put the old instructions under it with a heading like "Pre-0.7.1 Instructions"

Reply:
(Also remove the "In a future release" callout above)

Reply:
Just realized I misunderstood the scope of these docs! Looks good.